test/catalog.tests.bom

```diff
@@ -22,6 +22,11 @@ brooklyn.catalog:
         launch.command: |
           while true ; do sleep 3600; done & echo $! > ${PID_FILE}
         checkRunning.command: ""
+        brooklyn.config:
+          pre.install.command:
+            sudo yum install -y yum-plugin-fastestmirror
+            sudo yum update -y
```

Comment:
This seems to be required in some/most cases otherwise it won't install python2.7.  I originally had a PR (4) which did this in the cassandra node but it made the tests run so slowly that they were taking over an hour.
I could implement option 2 so that it is available on all cassandra nodes but not run it - that seems like a reasonable approach